test/functional/teacher_sets_controller_test.rb

```diff
@@ -13,39 +14,15 @@ class TeacherSetsControllerTest < ActionController::TestCase
     assert_not_nil assigns(:teacher_sets)
   end
 
-  test "should get new" do
-    get :new
+  test "should show teacher_set" do
+    get :show, params: { id: @teacher_set.id }
     assert_response :success
   end
 
   test "should create teacher_set" do
-    assert_difference('TeacherSet.count') do
-      post :create, teacher_set: { description: @teacher_set.description, slug: @teacher_set.slug, title: @teacher_set.title }
+    assert_no_difference 'TeacherSet.count' do
+      post :create, params: {teacher_set: {description: @teacher_set.description, title: @teacher_set.title}}
     end
-
-    assert_redirected_to teacher_set_path(assigns(:teacher_set))
-  end
-
-  test "should show teacher_set" do
-    get :show, id: @teacher_set
     assert_response :success
   end
-
-  test "should get edit" do
```

Comment:
Where did this go?

Reply:
@daryanypl In teacherset_controller 'edit' method is no there. So removed get method.